tests/lib/devicetree/api/app.overlay

```diff
@@ -431,6 +441,15 @@
 			interrupt-names = "err", "stat", "done";
 		};
 
+		/* there should only be one of these */
```

Comment:
The existing `test_irq` node can be extended to include `interrupts-extended`, without the need to create another binding file.

Reply:
Not really, if a node has both `interrupts` and `interrupts-extended` properties, the `interrupts-extended` property takes precedence and nothing is generated for the interrupts in `interrupts`. The test_irq node's compatible requires the `interrupts` property, so it can not be reused :)

Reply:
you are absolutely right